src/vuex/modules/keystore.js

```diff
@@ -35,8 +35,7 @@ export default () => {
       return getSeed()
     },
     async getAddressFromSeed(store, { seedPhrase, network }) {
```

Comment:
passing around the whole network object saves us from needing to search for the network in nested functions